src/test/resources/json/after-step-fails.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "id":"simpleId",
+        "name":"Simple feature",
+        "keyword":"Feature",
+        "elements":[
+            {
+                "keyword":"Scenario",
```

Comment:
I believe you can merge this into sample.json and select proper elements to assertion